README.md

```diff
@@ -4,4 +4,36 @@ This is the code behind https://crowdstrike.github.io/community/
 On that webpage, you can find content such as:
 - [List of CrowdStrike's Open Source Projects](https://crowdstrike.github.io/community/)
 - [Guidance for CrowdStrike Engineering](https://crowdstrike.github.io/community/open-source-policy/#engineering)
-- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
\ No newline at end of file
+- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
+
+
+
+## Developing the Site
+
+1. Ensure ``bundler`` and `jekyll` and ``rake`` are installed locally:
+```shell
+gem install bundler jekyll rake
+```
+2. Install NPM dependencies. The webroot is the `docs` folder, so you must change into that directory:
+```shell
+cd docs/ && npm install
+```
+
+3. Start the development server locally. It will watch for chances you make, and reload (nearly) instantly:
```

Comment:
```suggestion
3. Start the development server locally. Add the `--incremental` flag if you want to watch and reload changes:
```

Reply:
`bundle exec jekyll serve` automatically adds the ``--incremental`` flag

Reply:
From what I have seen, it doesn't get turned on automatically. Line 32 shows it as disabled.

Should we have instructions for linux vs macos?